addons/web/static/src/views/fields/datetime/datetime_field.js

```diff
@@ -47,6 +47,8 @@ export class DateTimeField extends Component {
         startDateField: { type: String, optional: true },
         warnFuture: { type: Boolean, optional: true },
         showSeconds: { type: Boolean, optional: true },
+        minPrecision: { type: String, optional: true },
+        maxPrecision: { type: String, optional: true },
```

Comment:
Hello @FlorianGilbert, this looks mostly good to me, the only thing I would change is add props validation to make it easier to understand what these props are waiting for without having to navigate to the hook itself. Something like this should be enough:

```js
minPrecision: {
    type: String,
    optional: true,
    validate:  e => ["days", "months", "years", "decades"].includes(e)
},
maxPrecision: {
    type: String,
    optional: true,
    validate:  e => ["days", "months", "years", "decades"].includes(e)
},
```

Reply:
Hello @BastienFafchamps 
Thank you for your review! 

I can do that, I was wondering if I may export the `PRECISION_LEVEL` constant [here](https://github.com/odoo/odoo/blob/06fe77a9af9e2f0ec0d2a0e3481305c646b006f6/addons/web/static/src/core/datetime/datetime_picker.js#L161) and use it like we did for the date picker?

https://github.com/odoo/odoo/blob/06fe77a9af9e2f0ec0d2a0e3481305c646b006f6/addons/web/static/src/core/datetime/datetime_picker.js#L302-L310

Maybe it's a bit too much? :thinking:

Reply:
Yeah this might be a bit much, I think the validation is fine now 👍

Reply:
Perfect, thank you :smile: